kratos/processes/calculate_embedded_nodal_variable_from_skin_process.h

```diff
@@ -592,6 +594,16 @@ class CalculateEmbeddedNodalVariableFromSkinProcess : public Process
                 }
             }
         }
+
+        rModelPart.Elements().Unique();
```

Comment:
```suggestion
```
This shouldn't be required, we're using a set to avoid having repeated edges (elements).